src/zzz_od/application/inventory_scan/ocr_worker.py

```
@@ -0,0 +1,146 @@
+        log.debug(f"OCR处理完成: 成功{self._processed_count}个, 失败{self._error_count}个")
+
+    def stop(self):
+        """停止工作线程"""
+        self.resume()
+        self._stop_event.set()
+        if self._thread:
+            self._thread.join(timeout=5)
+        log.debug("OCR工作线程已停止")
+
+    def pause(self):
+        """暂停处理新任务（队列保留）"""
```

Ruff (0.15.4)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)